qiskit/providers/models/pulsedefaults.py

```diff
@@ -56,17 +56,17 @@ class PulseDefaultsSchema(BaseSchema):
     """Schema for PulseDefaults."""
 
     # Required properties.
-    qubit_freq_est = List(Number(), required=True,
-                          validate=Length(min=1))
-    meas_freq_est = List(Number(), required=True,
-                         validate=Length(min=1))
-    buffer = Integer(required=True, validate=Range(min=0))
-    pulse_library = Nested(PulseLibraryItemSchema, required=True, many=True)
-    meas_kernel = Nested(MeasurementKernelSchema, required=True)
-    discriminator = Nested(DiscriminatorSchema, required=True)
+    qubit_freq_est = List(Number(), required=False,
+                          validate=Length(min=1), missing=None)
+    meas_freq_est = List(Number(), required=False,
+                         validate=Length(min=1), missing=None)
+    buffer = Integer(required=False, validate=Range(min=0), missing=0)
+    pulse_library = Nested(PulseLibraryItemSchema, required=True, many=True, missing=lambda: [])
+    meas_kernel = Nested(MeasurementKernelSchema, required=False, missing=None)
+    discriminator = Nested(DiscriminatorSchema, required=False, missing=None)
 
     # Optional properties.
```

Comment:
For consistency with rest of the codebase, can you remove `required=False` from the _optional_ parameters and place them under:
```
# Optional properties.
```
￼